src/profile-logic/process-profile.js

```diff
@@ -727,10 +727,10 @@ function _adjustMarkerTimestamps(
         return data;
       }
       const newData = immutableUpdate(data);
-      if ('startTime' in newData) {
+      if (typeof newData.startTime === 'number') {
         newData.startTime += delta;
       }
-      if ('endTime' in newData) {
+      if (typeof newData.endTime === 'number') {
```

Comment:
because `TracingMarkerMarkerPayload` isn't an exact object type, `typeof newData.endTime !== 'undefined'` isn't good enough, because flow would think it's an "unknown" type.